### ISSUE TITLE
Extend technical info with more troubleshooting information

### ISSUE THREAD

**Issue by KryoStoffer** [on-api/on-api#84]
First and second line support staff, need information that is not possible to represent with the current specification of technical_info, I propose to add these capabilities to technical info.

Detail about what I have found to be missing, can be seen in this PR: https://github.com/on-api/on-api/pull/83

**Comment by edruid** [comment]
The suggested extra data in #83 is:

```http
GET /api/2.4/tech/access/{accessId}/link/macaddresstable
```

| Parameter     | Type              | Description                                           |
|---------------|-------------------|-------------------------------------------------------|
| vlan          | integer           | Some services can exist on multiple vlans             |
| port          | string            | Distinguish port, In case uplink mac's are included   |
| learntAt      | string (iso8601)  | Some switches keep age of mac table entry             |

```http
GET /api/2.4/tech/access/{accessId}/link/leaseinfo
```

| Parameter                 | Type              | Description                           |
|---------------------------|-------------------|---------------------------------------|
| lastRenew                 | string (iso8601)  | Timestam of last renew                |

```http
GET /api/2.4/tech/access/{accessId}/link/status
```

| Parameter                                     | Type              | Description                                      |
|-----------------------------------------------|-------------------|--------------------------------------------------|
| fiber.biasTx                                  | string            | SFP TX bias current                              |
| fiber.supplyVoltage                           | string            | SFP Supply voltage                               |
| fiber.vendor                                  | string            | SFP Vendor                                       |
| fiber.partNumber                              | string            | SFP Part Number                                  |
| fiber.serialNumber                            | string            | SFP Serial Number                                |

**Comment by teemufrosen** [comment]
Working group meeting: PR will be updated: all fields are optional. Will be merged to 2.6.0.

**Comment by KryoStoffer** [comment]
Project handling too complicated.
